@kiva/kv-components/vue/KvMap.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+	<div
+		class="tw-relative tw-block tw-w-full"
+		:style="mapDimensions"
+	>
+		<div
+			:id="`kv-map-holder-${mapId}`"
+			:ref="refString"
+			class="tw-w-full tw-h-full tw-bg-black"
+			:style="{ position: 'absolute' }"
+		></div>
+	</div>
+</template>
+
+<script>
+import { createIntersectionObserver } from '../utils/observerUtils';
```

Comment:
Since we don't have existing utils in this project, you can just copy in these two methods (`createIntersectionObserver` and `testDelayedGlobalLibrary`) to this component for now. They aren't used elsewhere in this library as well.